etp-backend/src/main/clj/solita/etp/service/valvonta_kaytto/toimenpide.clj

```diff
@@ -83,11 +83,9 @@
                         :rfi-warning
                         :decision-order-hearing-letter
                         :decision-order-actual-decision
-                        :decision-order-notice-first-mailing
```

Comment:
Oliko tämä tiketiltä "Tiedoksianto: kirjaamo vie käsin prosessin eteenpäin AsHassa, ETP ei tee siirtoa"?

Reply:
Kyllä.